sequence_processing_pipeline/tests/test_FastQCJob.py

```diff
@@ -1,30 +1,13 @@
 import unittest
-from sequence_processing_pipeline.FastQCJob import FastQCJob
-from os.path import join
-from functools import partial
+import logging
 
 
-class TestFastQCJob(unittest.TestCase):
```

Comment:
Can you readd this tests and make sure it passes with your new code? Not OK to remove tests.